Libraries/CustomComponents/NavigationExperimental/NavigationHeader.js

```diff
@@ -133,11 +138,17 @@ class NavigationHeader extends React.Component<DefaultProps, Props, any> {
     });
 
     return (
-      <View style={[ styles.appbar, style ]} {...viewProps}>
+      <Animated.View style={[
+          styles.appbar,
+          { height: APPBAR_HEIGHT + this.props.statusBarHeight },
```

Comment:
Are you suggesting having a structure something like the following?

```
<View style={style.header}> // container
  <Animated.View style={{ height: this.props.statusBarHeight }} />
  <View style={style.appbar}>
    ...
  </View>
</View>
```

Would make sense, downside is that it's another view in every hierarchy (although maybe ReactAndroid would collapse them, not sure if it does that with `Animated.View`s), but probably the simplest way to animate the status bar size independently if that's important. Still don't have a use case for that myself though, so it's hard for me to judge how important it is.